NetCord/Rest/RestGuild.cs

```diff
@@ -102,4 +62,274 @@ int GetHighestRolePosition(PartialGuildUser user)
+    public ImageUrl? GetDiscoverySplashUrl(ImageFormat format = ImageFormat.Png) => DiscoverySplashHash is string hash ? ImageUrl.GuildDiscoverySplash(Id, hash, format) : null;
+
+    /// <summary>
+    /// <see langword="true"/> if the user is the owner of the <see cref="RestGuild"/>.
+    /// </summary>
+    /// <remarks>
+    /// Only available in objects returned from <see cref="RestClient.GetCurrentUserGuildsAsync(GuildsPaginationProperties?, RestRequestProperties?)"/>.
+    /// </remarks>
+    public virtual bool IsOwner => _jsonModel.IsOwner;
+
+    /// <summary>
+    /// The ID of the <see cref="RestGuild"/>'s owner.
+    /// </summary>
+    public ulong OwnerId => _jsonModel.OwnerId;
+
+    /// <summary>
+    /// Total permissions for the user in the <see cref="RestGuild"/> (excludes overwrites and implicit permissions).
+    /// </summary>
+    /// <remarks>
+    /// Only available in objects returned from <see cref="RestClient.GetCurrentUserGuildsAsync(GuildsPaginationProperties?, RestRequestProperties?)"/>.
+    /// </remarks>
+    public Permissions? Permissions => _jsonModel.Permissions;
+
+    /// <summary>
+    /// ID of the <see cref="RestGuild"/>'s AFK channel.
+    /// </summary>
+    public ulong? AfkChannelId => _jsonModel.AfkChannelId;
+
+    /// <summary>
+    /// How long in seconds to wait before moving users to the AFK channel.
+    /// </summary>
+    public int AfkTimeout => _jsonModel.AfkTimeout;
+
+    /// <summary>
+    /// <see langword="true"/> if the <see cref="GuildWidget"/> is enabled.
+    /// </summary>
+    public bool? WidgetEnabled => _jsonModel.WidgetEnabled;
+
+    /// <summary>
+    /// The ID of the channel that the <see cref="GuildWidget"/> will generate an invite to, or <see langword="null"/> if set to no invite.
+    /// </summary>
+    public ulong? WidgetChannelId => _jsonModel.WidgetChannelId;
+
+    /// <summary>
+    /// The <see cref="NetCord.VerificationLevel"/> required for the <see cref="RestGuild"/>.
+    /// </summary>
+    public VerificationLevel VerificationLevel => _jsonModel.VerificationLevel;
+
+    /// <summary>
+    /// The <see cref="RestGuild"/>'s <see cref="NetCord.DefaultMessageNotificationLevel"/>.
+    /// </summary>
+    public DefaultMessageNotificationLevel DefaultMessageNotificationLevel => _jsonModel.DefaultMessageNotificationLevel;
+
+    /// <summary>
+    /// The <see cref="RestGuild"/>'s <see cref="NetCord.ContentFilter"/>.
+    /// </summary>
+    public ContentFilter ContentFilter => _jsonModel.ContentFilter;
+
+    /// <summary>
+    /// A dictionary of <see cref="Role"/> objects indexed by their IDs, representing roles the <see cref="RestGuild"/>'s roles.
+    /// </summary>
+    public ImmutableDictionary<ulong, Role> Roles { get; set; }
+
+    /// <summary>
+    /// A dictionary of <see cref="GuildEmoji"/> objects, indexed by their IDs, representing the <see cref="RestGuild"/>'s custom emoji.
```

Comment:
```suggestion
    /// A dictionary of <see cref="GuildEmoji"/> objects, indexed by their IDs, representing the <see cref="RestGuild"/>'s custom emojis.
```

Reply:
Emoji is actually plural.

Reply:
I think both options are ok
https://www.grammarly.com/blog/emojis-emoji/